src/docs/perf/app-size.md

```diff
@@ -74,30 +74,48 @@ Built build/app/outputs/apk/release/app-arm64-v8a-release.apk (4.6MB).
 
 ## iOS
 
-To get a rough idea of how large your release IPA is,
-run the following:
+Create an [Xcode App Size Report][]
+by configurting the app version and build
+as described in the [iOS create build archive instructions][].
 
-```shell
-flutter build ios && tar -zcf build/app.ipa build/ios/iphoneos/Runner.app && ls -lh build/app.ipa
-```
+Then:
 
-The resulting IPA file for the `example/helloworld` app
-(as of this writing) is 8.3 MB.
+1. Select **Product > Archive** to produce a build archive.
+1. In the sidebar of the Xcode Organizer window, select your iOS app,
+   then select the build archive you just produced.
+1. Click **Distribute App**.
 
-```shell
--rw-r--r--  1 userName  primarygroup   8.3M Oct 25 13:47 build/app.ipa
+`Method of distribution`
+: Ad Hoc, Development, or Enterprise
+
+`App Thinning`
+: All compatible device variants
+
+`Rebuild from Bitcode`
+: Selected (available if bitcode is enabled on your project)
+
+`Strip Swift symbols`
+: Selected
+
+Sign and export the IPA. The exported directory contains
+`App Thinning Size Report.txt` with details about your projected
+application size on different devices and versions of iOS.
+
+The App Size Report for the default demo app
+(as of this writing) shows:
```

Comment:
nit, maybe just say as of May 2020

Reply:
I'll check out stable, regenerate, and say "The App Size Report for the default demo app in Flutter 1.17 shows:"